honestcash-v2/src/app/modules/editor/container/editor-container.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from '@angular/core';
+import {Logger} from '../../../core';
+import {environment} from '../../../../environments/environment';
+
+const log = new Logger('Editor');
```

Comment:
This needs to be injected.

Reply:
this was directly copied from editor.zip file that you thought was worth keeping so again, please come back when editor-container component is inside the editor root folder just like the welcome module has its container inside its root so you don't notice these inconsistencies but small stuff that are not yet mentioning because it is nowhere near done yet...

Reply:
LoggerService is not an Injectable class. I guess the point of this was to use this in non-angular environments. so when I look throughout the app, I did not introduce this LoggerService and whoever did, did it explicit construction of this with new Logger so if you want me to change every Logger in the app, please create a task for it then resolve this.